components/epaxos/src/instance/instance.rs

```diff
@@ -0,0 +1,32 @@
+use super::super::command;
+use super::super::replica;
+
+#[cfg(test)]
+#[path = "./tests/instance_tests.rs"]
+mod tests;
+
+/// protocol buffer serialized
+pub type InstanceNum = i64;
+pub struct InstanceID {
+    pub replica_id: replica::ReplicaID,
+    pub instance_num: InstanceNum,
+}
+
+pub enum InstanceStatus {
```

Comment:
```suggestion
pub enum Status {
```
在instance module了,去掉instance?

Reply:
这个还是要加上的，因为rust里引用数据结构的时候，前面一般不加module name，
直接写一个Status，一是容易冲突，二是不太容易读懂是什么的Status